scripts/openmc-ace-to-hdf5

```diff
@@ -0,0 +1,205 @@
+                    neutron = openmc.data.IncidentNeutron.from_ace(
+                        table, args.metastable)
+                except Exception as e:
+                    print('Failed to convert {}: {}'.format(table.name, e))
+                    continue
+
+                print('Converting {} (ACE) to {} (HDF5)'.format(table.name,
+                                                                neutron.name))
+
+                # Determine filename
+                outfile = os.path.join(args.destination,
+                                       neutron.name.replace('.', '_') + '.h5')
+                neutron.export_to_hdf5(outfile, 'w', libver=args.libver)
+
+                # Register with library
+                library.register_file(outfile)
+
+                # Add nuclide to list
+                nuclides[name] = outfile
+            else:
+                # Then we only need to append the data
+                try:
+                    neutron = \
+                        openmc.data.IncidentNeutron.from_hdf5(nuclides[name])
+                    print('Converting {} (ACE) to {} (HDF5)'
+                          .format(table.name, neutron.name))
+                    neutron.add_temperature_from_ace(table, args.metastable)
+                    neutron.export_to_hdf5(nuclides[name] + '_1', 'w',
+                                           libver=args.libver)
+                    os.rename(nuclides[name] + '_1', nuclides[name])
+                except Exception as e:
+                    print('Failed to convert {}: {}'.format(table.name, e))
+                    continue
+
+        elif xs.endswith('t'):
+            # Adjust name to be the new thermal scattering name
+            name = openmc.data.get_thermal_name(name)
+            # Thermal scattering data
+            if name not in nuclides:
+                try:
+                    thermal = openmc.data.ThermalScattering.from_ace(table)
+                except Exception as e:
+                    print('Failed to convert {}: {}'.format(table.name, e))
+                    continue
+                print('Converting {} (ACE) to {} (HDF5)'.format(table.name,
+                                                                thermal.name))
+
+                # Determine filename
+                outfile = os.path.join(args.destination,
+                                       thermal.name.replace('.', '_') + '.h5')
+                thermal.export_to_hdf5(outfile, 'w', libver=args.libver)
+
+                # Register with library
+                library.register_file(outfile)
+
+                # Add data to list
+                nuclides[name] = outfile
+
+            else:
+                # Then we only need to append the data
+                try:
+                    thermal = openmc.data.ThermalScattering.from_hdf5(
+                                                                 nuclides[name])
+                    print('Converting {} (ACE) to {} (HDF5)'
```

Comment:
Missing a space after the comma here `table.name,thermal.name` -> `table.name, thermal.name`